src/tree_data.h

```diff
@@ -1440,6 +1440,28 @@ int lyd_lyb_data_length(const char *data);
  */
 void *lyd_set_private(const struct lyd_node *node, void *priv);
 
+/**
+ * @brief Get the data node based on a simple XPath.
+ * This API returns the closest parent of the node (or the node itself)
+ * identified by the nodeid (path). This API uses lyd_find_sibling() If cache is
+ * enabled and the siblings are NOT top-level nodes, this function finds the
+ * node in a constant time.
+ * @param[in] data_tree Existing data tree to get.
+ * @param[in] ctx Context to use.
+ * @param[in] path Simple data path (see @ref howtoxpath). List nodes can have predicates, one for each list key
+ * in the correct order and with its value as well or using specific instance position, leaves and leaf-lists
+ * can have predicates too that have preference over \p value. When specifying an identityref value in a predicate,
+ * you MUST use the module name as the value prefix!
```

Comment:
Just copying the description from `lyd_new_path()` will not do because this is a subset of what `lyd_new_path()` is doing. List predicates are always mandatory and I would say it should also be true for leaf-lists. Leaves do not require value at all, I think, but please verify that. If so, then you can actually remove `value` and `value_type` parameters.

Reply:
Yes, you are correct. I have removed value and value_type parameters.
Please check the description again, If i am missing anything please let me know.